app/src/main/java/aashrai/android/gettowork/view/activity/SettingsActivity.java

```diff
@@ -49,6 +50,8 @@
   }
 
   private void configureSearch() {
+   search.setCompoundDrawables(null, null,
```

Comment:
Yes, good point. I should use VectorDrawableCompat instead of getDrawable used here.